testsuite/features/secondary/srv_scc_user_credentials.feature

```diff
@@ -0,0 +1,45 @@
+# Copyright 2017-2021 SUSE LLC
+# Licensed under the terms of the MIT license.
+
+Feature: SCC user credentials in the Setup Wizard
+# The purpose of this feature is not the same that the feature in core, 
+# this feature aimes to test the manipulation of scc credentials while 
+# the other in core aimes to setup scc credentials.
```

Comment:
This doesn't need to be a comment.

Reply:
Agree, just remove it.

Reply:
Remove as a comment and transform it as acceptance criteria. Just as we do in other features.

Reply:
Done